system.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// StopEtcd stops etcd on a specific host
+func StopEtcd(node TestbedNode) error {
```

Comment:
Yeah, I'd prefer this route too. I'll try to get to the build repo later this week to resolve those issues there.